lib/tasks/vacancies_to_update.yaml

```diff
@@ -212,3 +213,485 @@ We can offer:
+      <p>Applications are welcomed from both newly-qualified and experienced teachers.</p>
+      <p>Further details and an application pack, can be found on the council website www.newcastle.gov.uk or email: lilian.reeson@benfield.newcastle.sch.uk</p>
+      <p> Completed applications should be returned to Mrs L Reeson, PA to Headteacher, either by email or post.<br>
+      Email: lilian.reeson@benfield.newcastle.sch.uk<br>
+      Mrs L Reeson, Benfield School, Benfield Road, Newcastle, NE6 4NU </p>
+      <p> Please note CVs are not accepted.  <br>
+      Closing date: 9.00am Tuesday 8th May 2018<br>
+      Interviews: Week commencing 14th May 2018
+      </p>
+      <p>If you do not hear from us within six weeks of the closing date, you should assume that your application has not been successful.</p>
+      <p>At Benfield School we are committed to the safeguarding and welfare of young people. All posts are subject to an enhanced DBS check. We welcome applications from all sections of the community, regardless of gender, race, religion, disability, sexual orientation or age.</p>
+      <p>This post may involve working with children, vulnerable adults or dealing with sensitive information. Please note that in all cases written references will be taken up and made available to interviewers before the final selection stage; even if you indicate otherwise.</p>"
+      subject_name: 'Science'
+      starts_on: '01/09/2018'
+      min_pay_scale: 'Main pay range 1'
+      max_pay_scale: 'Upper pay range 3'
+    -
+      slug: 'support-assistants'
+      job_title: 'Support Assistants (2 roles)'
+      job_description: "<p><em> ‘Benfield is a good school that has improved considerably since its previous inspection’<br> ‘Teachers have good subject knowledge and use this to support learning well. They are enthusiastic and use some innovative techniques that enthuse students’<br> (Ofsted May 2015)</em></p>
+      <p>Exciting opportunities have arisen for confident and knowledgeable individuals to join our professional and highly skilled team. We are keen to employ enthusiastic Support Assistants for the following roles:
+      <ul><li> dedicated support for science department (must have studied A Level science)</li>
+      <li>general classroom support duties</li></ul></p>
+      Both roles are part time (32.5 hours over 5 days per week) and involve classroom support, individual and small group work in close collaboration with other school staff and supporting professionals. In return we offer the opportunity to work in an innovative and supportive environment.
+      <p> Application forms and further details are available on the council website<br>
+      www.newcastle.gov.uk and returnable by email to admin@benfield.newcastle.sch.uk<br>
+      or by post to Mrs L Reeson, Benfield School, Benfield Road, Newcastle NE6 4NU.</p>
+      <p>Please note CVs not accepted.<br>
+      Closing Date:            9.00am Tuesday 8th May 2018<br>
```

Comment:
Is this a field we are happy to put into the right place in the `ends_on` field?

Reply:
No. We don't capture the application closing date.